wkcuber/wkcuber/cubing.py

```diff
@@ -319,25 +319,28 @@ def cubing(
 
     logging.info("Found source files: count={} size={}x{}".format(num_z, num_x, num_y))
 
-    ensure_wkw(target_wkw_info)
-
     with get_executor_for_args(executor_args) as executor:
         job_args = []
         # We iterate over all z sections
         for z in range(start_z, num_z + start_z, BLOCK_LEN):
```

Comment:
Why is it enough to use the `block_len` (instead of `block_len*file_len`)?

Reply:
I think what you want to avoid is concurrent writes to the same block/bucket. Writing to the same wkw file should be fine, as long as the writes are in distinct buckets. So parallelizing over BLOCK_LEN should be ok. Does this make sense?

Reply:
Ah ok. I thought that concurrent writes to the same file are a problem.

Reply:
> Ah ok. I thought that concurrent writes to the same file are a problem.

It is only a problem, if the files are compressed, which should not be the case here.